components/members-list/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import MemberListItem from 'components/member-list-item';
+import Loader from 'components/loader';
+
+import useFetch from 'custom-hooks/useFetch';
+import classNames from './member-list.module.scss';
+
+const URL = 'https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/ids/mapping.json';
+
+const MembersList = () => {
+  let { loading, error, data } = useFetch(URL);
```

Comment:
at least first set can be server fetched, and later on the client fetched

Reply:
We are using Static-Build, we cannot use any server fetching features of Next.js.

Reply:
I forget, we not server-rendered.